run/views.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.decorators import api_view
+from rest_framework.parsers import JSONParser
+from .serializers import RunSerializer
+from .models import Run
+from rest_framework.response import Response
+
+
+@api_view(http_method_names=['POST'])
+def run(request):
+    runs_json = JSONParser().parse(request)
+    data_with_error = []
+    for run_json in runs_json:
+        serializer = RunSerializer(data=run_json)
+        if serializer.is_valid():
+            serializer.save()
+        else:
+            data_with_error.append(run_json)
+    return Response({'data_with_error': data_with_error})
+
+
+@api_view(http_method_names=['GET'])
+def report(request):
+    data = JSONParser().parse(request)
+    runs = Run.objects.all().query(end_time_gte=data['from_time'])
+    return Response(runs)
```

Comment:
It's better if we won't insert API logic to our models. What is returned from the run api should be handled in the run api. The current design requires you to return the id of the files.

Currently what you are returning is simply the whole data of an instance of Run. What we need is some thing like this:
```
{
   "run_token": "b204217b677944e7c8d2a12c670b5467",
   "execution_status": "success",
   "outputs": {
                         'name': 'game_log',
                         'file': b204217b677944e7c8d2a12c670b5467
    },
   "additional_info": {   }
}
```